docs/Homebrew-and-Python.md

```diff
@@ -2,18 +2,16 @@
 
 This page describes how Python is handled in Homebrew for users. See [Python for Formula Authors](Python-for-Formula-Authors.md) for advice on writing formulae to install packages written in Python.
 
-Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python.
-
-Homebrew provides formulae to brew Python 3.y. A `python@2` formula was provided until the end of 2019, at which point it was removed due to the Python 2 deprecation.
+Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python. On Linux Homebrew will install the necessary Python version that is needed to make your packages work.
 
 **Important:** If you choose to use a Python which isn't either of these two (system Python or brewed Python), the Homebrew team cannot support any breakage that may occur.
 
 ## Python 3.y
 
-Homebrew provides formulae for maintained releases of Python 3.y (`python@3.y`).
+Homebrew provides formulae for all maintained releases of Python 3.y (`python@3.y`).
 
 **Important:** Python may be upgraded to a newer version at any time. Consider using a version
-manager such as `pyenv` if you require stability of minor or patch versions for virtual environments.
+manager such as `pyenv` or `pipx` if you require stability of minor or patch versions for virtual environments.
```

Comment:
I'd rather we just picked one and suggested it here.

Reply:
I _think_ we can leave this at just `pyenv` -- `pipx` is a great tool, but (AFAIK) doesn't perform Python version management itself.

Reply:
Fun fact, `pipx` is gaining some version management features in the next release: https://github.com/pypa/pipx/pull/1243

But agreed this can just mention `pyenv` as it's the current standard